scripts/circle-ci-tests.sh

```diff
@@ -32,9 +32,12 @@ function run_paver_quality {
 function test_system() {
     system=${1}
     shift
-    flags='--cov-args="-p"'
-    # paver test_system -s "${system}" -t "${test_id}" ${flags}
-    DJANGO_SETTINGS_MODULE=${system}.envs.test pytest ${@}
+    random_flag=''
+    if [ "${system}" == "lms" ]; then
+        random_flag="--randomly-dont-reorganize"
+    fi
+    echo "DJANGO_SETTINGS_MODULE=openedx.stanford.${system}.envs.test pytest ${random_flag} ${@}"
+    DJANGO_SETTINGS_MODULE=openedx.stanford.${system}.envs.test pytest ${random_flag} ${@}
```

Comment:
Yeah.... we'll have better luck if we run w/ `stanford` settings...

Reply:
Is there no way to make this change without forking as much?

Reply:
Errr, rather, this file _may_ still exist, but edx never upgraded to circle2.0;
they aren't using it.

Reply:
Also please remember, we've heavily modified this file for years now...